interfaces/console/src/app/console/layout.tsx

```diff
@@ -46,9 +48,9 @@ export default function ConosleLayout({
     setSnackbarMessage,
     subscriptionClient,
   } = useAppContext();
-  const [notifications, setNotifications] = useState<
-    NotificationsResDto[] | []
-  >([]);
+  const router = useRouter();
+  const [notifications, setNotifications] = useState<any>([]);
+
```

Comment:
Why we change `notifications` type define from `NotificationsResDto[]` to `any`?

Reply:
Fixed